Configurateur/front/lib/screens/container-creation/maps_screen.dart

```diff
@@ -0,0 +1,175 @@
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:front/components/custom_app_bar.dart';
+import 'package:front/components/progress_bar.dart';
+import 'package:front/network/informations.dart';
+import 'package:front/services/http_service.dart';
+import 'package:front/services/storage_service.dart';
+import 'package:go_router/go_router.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+import 'package:geolocator/geolocator.dart';
+
+class MapsScreen extends StatefulWidget {
+  const MapsScreen(
+      {super.key,
+      this.lockers,
+      this.amount,
+      this.containerMapping,
+      this.id,
+      this.container});
+
+  final String? lockers;
+  final int? amount;
+  final String? containerMapping;
+  final String? id;
+  final String? container;
+
+  @override
+  State<MapsScreen> createState() => MapsState();
+}
+
+class MapsState extends State<MapsScreen> {
+  String jwtToken = '';
+  final Completer<GoogleMapController> _controller =
+      Completer<GoogleMapController>();
+
+  static CameraPosition _kGooglePlex = const CameraPosition(
+    target: LatLng(47.210528, -1.566956),
```

Comment:
Where is it ? Just add a comment for comprehension

Reply:
epitech nantes